_saas-integrations/netsuite/v1/netsuite-v1.md

```diff
@@ -0,0 +1,426 @@
+         - **Role**: Select the Stitch role you created in [Step 4](#create-stitch-netsuite-role).
+         - **Token Name**: Enter a name for the token. For example: `Stitch`
+      4. Click the **Save** button. The confirmation page will display a **Token ID and Secret**.
+      5. **Copy the Token ID and Secret somewhere handy.** You'll need these credentials to complete the setup in Stitch.
+
+  - title: "Locate your {{ integration.display_name }} Account ID"
+    anchor: "locate-netsuite-account-id"
+    content: |
+      {% include layout/inline_image.html type="right" file="integrations/netsuite-account-id.png" alt="" max-width="250px" %}
+      
+      1. Using the global search, type `page: web services` and click the **Page: Web Services Preferences** result.
+      2. In the Primary Information section, locate the **Account ID** field as shown in the image on the right.
+      
+      **Note**: If your Account ID contains a suffix - `1234567_SB2`, for example - it should be included when entering the ID into Stitch.
+
+  - title: "add integration"
+    content: |
+      4. In the **Account** field, enter the {{ integration.display_name }} account ID you retrieved in [Step 7](#locate-netsuite-account-id).
+      5. In the **Consumer Key** field, paste the Consumer Key you generated when you [created Stitch's integration record](#create-stitch-integration-record).
+      6. In the **Token ID** field, paste the Token ID you generated when you [created Stitch's access tokens](#create-access-tokens).
+      7. In the **Consumer Secret** field, paste the Consumer Secret you generated when you [created Stitch's integration record](#create-stitch-integration-record).
+      8. In the **Token Secret** field, paste the Token Secret you generated when you [created Stitch's access tokens](#create-access-tokens).
+  - title: "historical sync"
+  - title: "replication frequency"
+  - title: "track data"
+
+
+# -------------------------- #
+#     Replication Details    #
+# -------------------------- #
+
+replication-sections:
+  - content: |
+      In this section:
+
+      {% for section in page.replication-sections %}
+      - [{{ section.title | flatify }}](#{{ section.anchor }})
+      {% endfor %}
+
+  - title: "Custom records"
+    anchor: "custom-records"
+    content: |
+      For each custom record type in {{ integration.display_name }}, a table for that custom record type will be available for selection in Stitch.
+
+    subsections:
+      - title: "Table names for custom record types"
+        anchor: "custom-record-types-table-names"
+        content: |
+          Custom record tables are named `customrecord_[custom_record_type]`, where `[custom_record_type]` is the name of the custom record in {{ integration.display_name }}. For example: If a custom record were named `promo discount` in {{ integration.display_name }}, the corresponding table for those records would be named `customrecord_promo_discount`.
```

Comment:
It might be worth it to call out that the name here comes from the ID field in the custom record setup screen in NetSuite. If they don't put an ID in, it'll be something like `customrecord123`, since NetSuite just autoassigns a numerical ID.

Reply:
Done!